applications/CoSimulationApplication/python_scripts/solver_wrappers/sdof/static_sdof_solver_wrapper.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics as KM
+import KratosMultiphysics.CoSimulationApplication as KMC
+
+# Importing the base class
+from KratosMultiphysics.CoSimulationApplication.base_classes.co_simulation_solver_wrapper import CoSimulationSolverWrapper
+
+# Other imports
+from .sdof_static_solver import SDoFStaticSolver
+
+def Create(settings, solver_name):
+    return SdofStaticSolverWrapper(settings, solver_name)
+
+class SdofStaticSolverWrapper(CoSimulationSolverWrapper):
```

Comment:
I will talk to Andreas how to only have one wrapper for both sdof solvers
I will let you know